ExchangeActiveSync/Source/MS-ASCAL/TestSuite/MS-ASCAL_RequirementSpecification.xml

```diff
@@ -6132,6 +6546,15 @@ AirSync [and reference is] [MS-ASCMD] section 2.2.2.19.</ns1:Description>
 		<ns1:IsNormative>Normative</ns1:IsNormative>
 		<ns1:Verification>Test Case</ns1:Verification>
 	</ns1:Requirement>
+	<ns1:Requirement>
+		<ns1:REQ_ID>MS-ASCAL_R2175001</ns1:REQ_ID>
+		<ns1:Doc_Sect>2.2.2.36</ns1:Doc_Sect>
+		<ns1:Description>[In OrganizerName] [When protocol version 16.1 is used, the client MUST NOT include the OrganizerName element in command requests and] the server will use the name of the current user.</ns1:Description>
+		<ns1:Behavior>Protocol</ns1:Behavior>
+		<ns1:Scope>Server</ns1:Scope>
+		<ns1:IsNormative>Normative</ns1:IsNormative>
+		<ns1:Verification>Test Case</ns1:Verification>
```

Comment:
R2175001 is adapter

Reply:
R2175 is verified in test case, should we update the test case R2175001 base on it?

Reply:
R2175 is verified in test case, should we update the test case R2175001 base on it?